src/benchmarks/micro/libraries/Microsoft.Extensions.DependencyInjection/ActivatorUtilitiesBenchmark.cs

```diff
@@ -8,6 +8,77 @@
 
 namespace Microsoft.Extensions.DependencyInjection
 {
+    [BenchmarkCategory(Categories.Libraries)]
+    public class AA
```

Comment:
NIT: A meaningful name could be useful

Reply:
Removed this; it was a temp benchmark to help diagnose an issue.